en/02_Developer_Guides/08_Performance/01_Caching.md

```diff
@@ -154,6 +154,10 @@ you need to be careful with resources here (e.g. filesystem space).
 
 
 ```yml
+---
+Name: my-project-cache
+After: '*#corecache'
```

Comment:
```suggestion
After: '#corecache'
```
The wildcard is implicit - no need to include it in the example as it may lead people to think there's more going on than there actually is.

Reply:
Thanks. TBH I didn't know what the asterisk or `#` did. Just as a sanity check - these are all equivalent?

```
After:
  - 'corecache'
```
```
After: 'corecache'
```
```
After: '#corecache'
```
```
After: 'framework#corecache'
```

Reply:
Technically that last one would only be before a config section named `corecache` in a framework module, but otherwise yes.